### PR TITLE
Add connection creation time and lifetime

### PR DESCRIPTION
**Pull request by Danielybl**
Add option --detail-conn to print lifetime and creation time for connections.
The sample output is listed as below.

ipvsadm -lnc --detail-conn
[11]tcp 30s    28s    2019-02-21 13:24:05  SYN_RECV    10.2.211.140:55819 10.10.76.66:80     10.10.127.71:1032  10.10.127.67:8000
[12]tcp 30s    27s    2019-02-21 13:24:05  SYN_RECV    10.2.211.140:55818 10.10.76.66:80     10.10.127.70:1033  10.10.127.67:8000

### CODE REVIEW

File: tools/ipvsadm/ipvsadm.c [hunk 1]
```diff
@@ -245,6 +246,7 @@ static const char commands_v_options[NUMBER_OF_CMD][NUMBER_OF_OPT] =
 #define FMT_PERSISTENTCONN	0x0020
 #define FMT_NOSORT		0x0040
 #define FMT_EXACT		0x0080
+#define FMT_DETAILCONN  0x0100
 
 #define SERVICE_NONE		0x0000
 #define SERVICE_ADDR		0x0001
```
Comment (ywc689) [inline tools/ipvsadm/ipvsadm.c]: fix indentation

File: tools/keepalived/keepalived/libipvs-2.6/libipvs.h [hunk 2]
```diff
@@ -44,7 +44,8 @@
 #define OPT_SIPHASH		0x10000000
 #define OPT_QIDHASH		0x20000000
 #define OPT_HASHTAG		0x40000000
-#define NUMBER_OF_OPT		31
+#define OPT_DETAILCONN  0x80000000
+#define NUMBER_OF_OPT		32
```
Comment (ywc689) [inline tools/keepalived/keepalived/libipvs-2.6/libipvs.h]: fix indentation

File: src/ipvs/ip_vs_conn.c [hunk 3]
```diff
@@ -759,9 +762,7 @@ struct dp_vs_conn *dp_vs_conn_new(struct rte_mbuf *mbuf,
     rte_atomic32_set(&new->refcnt, 1);
     new->flags  = flags;
     new->state  = 0;
-#ifdef CONFIG_DPVS_IPVS_STATS_DEBUG
-    new->ctime = rte_rdtsc();
-#endif
+    new->ctime = rte_rdtsc_precise();
```
Comment (ywc689) [inline src/ipvs/ip_vs_conn.c]: I found the time cost of "rte_rdtsc" is a bit high when I did performance test using perf.  Do we really need to know exactly when the connection created?

**Comment by ywc689** [comment]
Please fix the code conflicts. We'll accept it later.

File: src/ipvs/ip_vs_conn.c (hunk 3 above)
Comment (Danielybl) [reply src/ipvs/ip_vs_conn.c]: Yes, agree. I find this command is not very useful for debugging. I think we can keep this out first.

**Pull request closed without merging**